packages/material-ui/src/ExpansionPanelActions/ExpansionPanelActions.js

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import clsx from 'clsx';
+import withStyles from '../styles/withStyles';
+
+export const styles = {
+  /* Styles applied to the root element. */
+  root: {
+    display: 'flex',
+    alignItems: 'center',
+    padding: 8,
+    justifyContent: 'flex-end',
+  },
+  /* Styles applied to the root element if `disableSpacing={false}`. */
+  spacing: {
+    '& > :not(:first-child)': {
+      marginLeft: 8,
+    },
+  },
+};
+
+const ExpansionPanelActions = React.forwardRef(function ExpansionPanelActions(props, ref) {
+  if (process.env.NODE_ENV !== 'production') {
+    console.error(
+      [
+        'Material-UI: the ExpansionPanelActions component was renamed to AccordionActions to match the naming convention of the community.',
+        '',
+        "You should use `import { AccordionActions } from '@material-ui/core'`",
+        "or `import AccordionActions from '@material-ui/core/AccordionActions'`",
+      ].join('\n'),
+    );
```

Comment:
One warning in packages/material-ui/src/ExpansionPanel/ExpansionPanel.js should be enough. I believe people can't use it without the wrapping component.

Reply:
However, I think that you were right, it can help to have a warning down to the last wrong import. I have changed my mind.

Reply:
Thinking back to typography v2 I would consider deduplicating it. Once you're aware it should be pretty straightforward to fix every warning since grepping by component name wouldn't include too much noise.

Reply:
What about a `warnOnce` flag to have deduplication?

Reply:
Sorry I am not following, do we want to remove the warnings from the other components?

Reply:
It's somewhat related to #15343. @mnajdova what do you think of ee388b90253e5649d02f1e0d3e8f2e861c2489bd?

Reply:
Ah got it, it makes sense, I can see that otherwise it would be too much noise 👍